app/players/elves/druid.py

```diff
@@ -0,0 +1,22 @@
+from app.players.elves.elf import Elf
+
+
+class Druid(Elf):
+    def __init__(self,
+                 nickname: str,
+                 musical_instrument: str,
+                 favourite_spell: str) -> None:
+        super().__init__(nickname, musical_instrument)
+        self._favourite_spell = favourite_spell
+
+    def play_elf_song(self) -> None:
+        instrument = self._musical_instrument
+        print(f"{self.nickname} is playing a song on the {instrument}")
+
+    def player_info(self) -> str:
+        nick = self.nickname
+        spell = self._favourite_spell
+        return f"Druid {nick}. {nick} has a favourite spell: {spell}"

```

Comment:
you don't need these variables as well, please fix it everywhere

Reply:
It was a flake8 remark. Line had more than 79 symbols